uktrade/data-hub-frontend
# Axios - Replace feature flag unit test

## Description of change

This replaces the unit tests around feature flags with functional tests. As this is the second piece of tests around middleware I have also refactored the naming and folder structure for layout testing. 

## Test instructions

Nothing should change so new functional tests should pass, as I have created mock routes. These new routes should only be accessible if you are NOT in production, if you try and access these routes in production you will get a 404.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
